drivers/sensors/lepton.c

```diff
@@ -493,10 +493,45 @@ static int snapshot(omv_csi_t *csi, image_t *image, uint32_t flags) {
     return 0;
 }
 
+static int config(omv_csi_t *csi, omv_csi_config_t config) {
+    if (config == OMV_CSI_CONFIG_INIT) {
+        // Extra delay after power-on
+        mp_hal_delay_ms(1000);
```

Comment:
This is okay because it's called only once, after power-on, but I don't think the extra 1s delay in reset is needed. It seems to work fine without it.